README.md

```diff
@@ -45,33 +15,17 @@ are organized. So, for example, you can ask questions like:
 * What Redis master and slave services are part of this Redis instance?
 * What pods are part of this Redis slave?
 
-Because DM stores its state in the cluster, not on your workstation, you can ask
-those questions from any client at any time.
-
-Templates live in ordinary Github repositories called template registries. See
```

Comment:
Seems like we should say something about registries here, and probably point to kubernertes/charts as the default shared public registry.